tools/calculte/encrypt.go

```diff
@@ -0,0 +1,17 @@
+package calculte
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+)
+
+func Md5Sum(str string) string {
+	h := md5.New()
+	h.Write([]byte(str))
+	return hex.EncodeToString(h.Sum(nil))
+}
+
+func GetSaltingPwd(passwd string) string {
```

Comment:
we would better define it in used place, password may hash by other algorithms in different places